src/CLI/clitree/cli-xml/ipv6.xml

```diff
@@ -24,6 +24,91 @@ limitations under the License.
     xsi:schemaLocation="http://www.dellemc.com/sonic/XMLSchema
                         http://www.dellemc.com/sonic/XMLSchema/clish.xsd">
 
+<VIEW name="enable-view">
+    <COMMAND
+        help="show IPv6 commands"
```

Comment:
Good to have help string to start with CAPS, just like how it was followed in os10

Reply:
ok